### DISCUSSION TITLE
How to pass session token to vaex.open

### DISCUSSION THREAD

**Discussion by Hasham04** [vaexio/vaex#1935] [Q&A] [answered]
Hello Friendly coder,

Vaex.open uses s3fs under the hood you can simply pass
fs_options={
'access_key':',
'secret_key':'',
'session_token'':
}

Can we kindly put this in the documentation that session_token is a valid argument for fs_options?

**Comment by Hasham04** [comment, marked as answer]
This question is answered.

**Comment by JovanVeljanoski** [comment]
PRs are welcome
